### PR TITLE
PMM-11000 Doc PMM Pacemaker EC2

### PR DESCRIPTION
**Pull request by anton-bystrov**
PMM-11000 Documentation how to install HA PMM instance via Pacemaker on EC2 AWS.

### CODE REVIEW

File: docs/how-to/pmm-ec2.md [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+
+At this article we provide information and procedures for configuring HA cluster, based on a Centos 9 Stream on Amazon Web Services using EC2 instances as cluster nodes.
+
+## Prerequisites
+
+- Two EC2 AWS Nodes built on the [Nitro System](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-types.html#ec2-nitro-instances)
+- Shared storage attached to both nodes (visible on both nodes as a block device, /dev/nvme1n1 in our example)
+- Shared network
+- Shared NFS mount (/mnt/nfs_share) to share configuration files
+- Allowed incoming/ougoing data in security groups for xpecific ports [Pacemaker](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/high_availability_add-on_reference/s1-firewalls-haar) and [PMM](https://docs.percona.com/percona-monitoring-and-management/setting-up/server/network.html), [Glusterfs](https://docs.gluster.org/en/latest/Administrator-Guide/Setting-Up-Clients/).
```
Comment (atymchuk) [inline docs/how-to/pmm-ec2.md:15]:
```suggestion
- Allow ingress/egress on specific ports in security groups [Pacemaker](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/high_availability_add-on_reference/s1-firewalls-haar) and [PMM](https://docs.percona.com/percona-monitoring-and-management/setting-up/server/network.html), [Glusterfs](https://docs.gluster.org/en/latest/Administrator-Guide/Setting-Up-Clients/).
```

File: docs/how-to/pmm-ec2.md [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+
+
+## Preparing environments
+
+This instructions should be done on both nodes.
```
Comment (atymchuk) [inline docs/how-to/pmm-ec2.md:20]:
```suggestion
Steps outlines in this instruction should be performed on both nodes.
```

File: docs/how-to/pmm-ec2.md [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+10.178.0.66 pmm1
+```
+
+## Installing AWS CLI on nodes
+
+For next steps you should install and configure AWS CLI. This instructions should be done on both nodes.
+
+The first install AWS CLI full instruction available [here](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html).
+
+The next define AWS credentials:
```
Comment (atymchuk) [inline docs/how-to/pmm-ec2.md:63]:
```suggestion
Next, define AWS credentials:
```

File: docs/how-to/pmm-ec2.md [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+::1         localhost localhost.localdomain localhost6 localhost6.localdomain6
+10.178.0.215 pmm2
+10.178.0.66 pmm1
+```
+
+## Installing AWS CLI on nodes
+
+For next steps you should install and configure AWS CLI. This instructions should be done on both nodes.
+
+The first install AWS CLI full instruction available [here](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html).
```
Comment (atymchuk) [inline docs/how-to/pmm-ec2.md:61]: Too vague of a sentence, not clear what the user should do.

**Review by atymchuk — CHANGES_REQUESTED** [review]

**Comment by rnovikovP** [comment]
Lovely and exciting content, but we can't put it in the official documentation for now. Let's start with the blog post, see how many people are interested, and then, if needed, find a way to improve UX to do such deploy.

**Comment by ademidoff** [comment]
Closing as stale.

**Pull request closed without merging**